src/app/genes/gene-overview/evidence-menu/rna/gene-network/gene-network.component.html

```diff
@@ -26,15 +26,15 @@
                 <div class="dot first"></div><div class="legend-text">2-3 Edges</div>
                 <div class="dot second"></div><div class="legend-text">4-5 Edges</div>
                 <div class="dot third"></div><div class="legend-text">6-7 Edges</div>
-                <div class="dot fourth"></div><div class="legend-text">8-9 Edges</div>
+<!--                <div class="dot fourth"></div><div class="legend-text">8-9 Edges</div>-->
             </div>
         </div>
         <ng-template #noDataImage>
-            <div class="col-md-8 col-sm-8 col-12 network-container no-data-vertical">
+            <div class="col-12 network-container no-data-vertical">
```

Comment:
Does this look right on medium and small screens?  Before you had the expected 8+4 columns (bootstrap grid is a 12 column layout), not it looks like you have a floating 4 column width div below.

Reply:
It looks fine to me at medium and small. Note this change was originally made by MattC and was merged and somehow got reverted.